webservices/common/models/itemized.py

```diff
@@ -122,6 +133,91 @@ class ScheduleA(BaseItemized):
     pdf_url = db.Column(db.String)
 
 
+class ScheduleAEfile(BaseRawItemized):
+    __tablename__ = 'real_efile_sa7'
+
+    file_number = db.Column("repid", db.Integer, index=True, primary_key=True)
+    related_line_number = db.Column("rel_lineno", db.Integer, primary_key=True)
+    committee_id = db.Column("comid", db.String, doc=docs.COMMITTEE_ID)
+    contributor_prefix = db.Column('prefix', db.String)
+    """
+    contributor = db.relationship(
+        'CommitteeHistory',
+        primaryjoin='''and_(
+                foreign(ScheduleA.contributor_id) == CommitteeHistory.committee_id,
+                ScheduleA.report_year + ScheduleA.report_year % 2 == CommitteeHistory.cycle,
+            )'''
+    )
+    """
+    #contributor_name = db.Column('contbr_nm', db.String, doc=docs.CONTRIBUTOR_NAME)
```

Comment:
Do we not need these relationships for the efiling version of the Schedule A data?  What about enabling downloads on the data, wouldn't we want to see the committee information like we do for the regular Schedule A data?  Or if it's not needed, should this be removed?